pgo/cmd/update.go

```diff
@@ -129,19 +129,24 @@ var UpdateUserCmd = &cobra.Command{
 	Short: "Update a postgres user",
 	Long: `UPDATE allows you to update a pgo user. For example:
 		//change a password
-		pgo update user mycluster --username=someuser --password=foo --update-passwords --expired=200
+		pgo update user mycluster --username=someuser --password=foo --valid-days=40
```

Comment:
Change the comment above this for clarity, something like " change a password, set valid days for 40 days from now"

Reply:
Updated.